### PR TITLE
Add Japanese (ja) locale

### PR DESCRIPTION
**Pull request by yuryu**
## Proposed Changes

* Add a Japanese (ja) translation
To make the service accessible to Japanese-speakers.

**Release Note**

```release-note
- Japanese (ja) translation
```

### CODE REVIEW

File: internal/i18n/locales/ja/default.po [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+#
+# navigation menus
+# ----------------
+
+msgid "nav.issue-code"
+msgstr "コードを発行""
```
Comment (mikehelmick) [inline internal/i18n/locales/ja/default.po:14]: seems like there might be an extra `"` here

Comment (yuryu) [reply internal/i18n/locales/ja/default.po]: thanks, fixed

**Review by mikehelmick — APPROVED** [review]

**Comment by sethvargo** [comment]
Thanks Emma!

**Pull request merged**